<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtOknoGlowneClass</class>
 <widget class="QMainWindow" name="QtOknoGlowneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtOknoGlowne</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listaPlyt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>256</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nowaPlyta">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usunPlyte">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtOknoGlowne.qrc"/>
 </resources>
 <connections/>
</ui>
